README.md

```diff
@@ -16,29 +16,32 @@ This project aims to build the official site for the community page so that any
 
 ## Tech Stack 🗃
 
- <img src="https://img.shields.io/badge/-MongoDB-yellow?style=flat&logo=mongoDB"> <img src="https://img.shields.io/badge/-expressJS-yellowgreen"> <img src="https://img.shields.io/badge/-ReactJS-%2300bfff%20">  <img src="https://img.shields.io/badge/-NodeJS-black?style=flat&logo=node.js">  
+ <img src="https://img.shields.io/badge/-MongoDB-yellow?style=flat&logo=mongoDB"> <img src="https://img.shields.io/badge/-expressJS-yellowgreen"> <img src="https://img.shields.io/badge/-ReactJS-%2300bfff%20">  <img src="https://img.shields.io/badge/-NodeJS-black?style=flat&logo=node.js"> 
+ 
+##  Getting Started 👨‍💻
+
+1. `git clone`
```

Comment:
git clone is not the right action to perform for getting started
One should fork the repository first.
Please make the changes accordingly.

Reply:
then after forking can I add `git clone`

Reply:
Yes, but `git clone` is not a complete command.
Please update it accordingly

Reply:
okay